docs/csi_ug_config_advanced_importvol.md

```diff
@@ -0,0 +1,177 @@
+# Importing an existing volume
+
+Use this information to import volumes created externally from the IBM® block storage CSI driver by using a persistent volume (PV) yaml file.
+
+Before starting to import an existing volume, find the following information in the existing volume in order to include the information in the persistent volume (PV) yaml file:
+
+-   `volumeHandle`
+-   `volumeAttributes` (optional)
+
+    Including:
+
+    -   `pool_name`:_<Name of Pool where volume is located>_ (Listed as `pool_id` for DS8000® Family systems.)
+    -   `storage_type`: <SVC | A9K | DS8K>
+    -   `volume_name`:_<Volume name>_
+    -   `array_address`:_<Array address>_
+
+To find the `volumeHandle`, use one of the following procedures:
+
+-   **Through command line (for Spectrum Virtualize Family):**
+
+    `    lsvdisk <volume name> | grep vdisk_UID`
+
+    ```screen
+    lsvdisk vol0 | grep vdisk_UID
+    vdisk_UID 600507640082000B08000000000004FF
+    ```
+
+-   **Through command line (for FlashSystem A9000 and A9000R):**
+
+    `vol_list_extended vol=<volume_name>`
+    
+
+    For example, for vol1:
+
+    ```screen
+    A9000>> vol_list_extended vol=vol1
+    Name   WWN                                Product Serial Number     
+    vol1   6001738CFC9035E8000000000091F0C0   60035E8000000000091F0C0 
+    ```
```

Comment:
These lines are inside a box. I suggest putting only the lower section inside a box.